hashin.py

```diff
@@ -388,17 +388,39 @@ def amend_requirements_content(requirements, all_new_lines):
     padding = " " * 4
 
     def is_different_lines(old_lines, new_lines, indent):
+        # This regex is used to only temporarily normalize the names of packages
+        # in the lines being compared. This results in "old" names matching
+        # "new" names so that hashin correctly replaces them when it looks for
+        # them.
+        rex = re.compile("[-_]")
```

Comment:
Shouldn't that be...
```suggestion
        rex = re.compile(r"[-_]")
```
A bit surprised that `flake8` didn't complain about that. 

Also, `rex` isn't a great name. It sounds like a pun on "regular expression" and it's not mentioning anything about its purpose in life or its use.